app/helpers/state_file/nj_homeowner_eligibility_helper.rb

```diff
@@ -0,0 +1,19 @@
+module StateFile
+  module NjHomeownerEligibilityHelper
+    class << self
+
+      def determine_eligibility(intake)
+        is_ineligible = intake.homeowner_home_subject_to_property_taxes == 'no' || (intake.homeowner_main_home_multi_unit == 'yes' && intake.homeowner_main_home_multi_unit_max_four_one_commercial == 'no')
+        is_unsupported = intake.homeowner_more_than_one_main_home_in_nj == 'yes' || intake.homeowner_shared_ownership_not_spouse == 'yes' || (intake.homeowner_main_home_multi_unit == 'yes' && intake.homeowner_main_home_multi_unit_max_four_one_commercial == 'yes')
+
+        if is_ineligible
```

Comment:
[PEBBLE] I think these return values would be good candidates to turn into constants, to reduce the risk that someone mis-types a string when checking the output of this function. i.e. at the top of this class you could add something like
```
Ineligible = :ineligible
Unsupported = :unsupported
Advance = :advance
```
and elsewhere, when checking output, you could do things like `if ...determine_eligibility(intake) == NjHomeownerEligibilityHelper::Unsupported`. a bit verbose, but less error-prone